src/es/overlay/uart.md

```diff
@@ -15,7 +15,7 @@ pin:
     direction: entrada
     active: alto (encendido)
 -->
-#UART - Universal Asynchronous Receiver/Transmitter (Receptor/Transmisor Universal Asíncrono)
+#UART - Universal Asynchronous Receiver/Transmitter (Receptor/TransSDIr Universal Asíncrono)
```

Comment:
Here's the other occurence of find-and-replace typos. At a quick glance seems like these are the only two though.